examples/bcic_iv_2a_cropped.py

```diff
@@ -14,61 +14,25 @@
 
 import torch
 from torch import optim
-from torch.utils.data import Dataset
```

Comment:
@robintibor can you name the examples with plot_ so they get rendered in the doc.

See for https://194-232335424-gh.circle-artifacts.com/0/dev/auto_examples/bcic_iv_2a_cropped.html#sphx-glr-auto-examples-bcic-iv-2a-cropped-py

it contains a hard coded link to file you are the only one to have. Can you relay on some moabb dataset?

Reply:
This was a thing we discussed briefly in Paris I think. There are examples (right now, the two not with `plot_` at the start) that are meant for reproducing an actual pipeline from a paper and may require GPU/longer computing times. As far as I remember, you suggested either change to having parameters that allow fast runtimes and some hint that they need to be changed to get good results or remove `plot_` to not have them rendered. I opted for second choice now. Still, of course would be better to fetch the dataset without local folder path etc. Would prefer to have this not block current PR merge, so created separate issue: https://github.com/braindecode/braindecode/issues/61

Reply:
I would see if we can rather run it with very little number of epochs (maybe just 1) to allow checks it runs and tell clearly on the top what to change to get fully reproducible results.